sklearn/metrics/classification.py

```diff
@@ -1875,7 +1878,8 @@ def log_loss(y_true, y_pred, eps=1e-15, normalize=True, sample_weight=None,
     y_true : array-like or label indicator matrix
         Ground truth (correct) labels for n_samples samples.
 
-    y_pred : array-like of float, shape = (n_samples, n_classes) or (n_samples,)
+    y_pred : array-like of float, \
+        shape = (n_samples, n_classes) or (n_samples,)
```

Comment:
I'd rather you reverted this change. But if you keep it, please indent this so it is not visually confused with the next line

Reply:
this was requested by Andy:  https://github.com/scikit-learn/scikit-learn/pull/12210#discussion_r221435338
but, I can change it back if that is preferred.